### PR TITLE
fix: popular feed

### PR DESCRIPTION
**Pull request by idoshamun**
The popular feed configuration was broken for a while but the issue was surfaced when we introduced caching.

This PR fixes it and also includes blocking the tags and sources on this feed.

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

File: src/integrations/feed/generators.ts [hunk 1]
```diff
@@ -111,22 +110,15 @@ export const feedGenerators: Record<FeedVersion, FeedGenerator> = Object.freeze(
...
-            remove_engaged_posts: true,
-            page_size_fraction: 1,
-            fallback_provider: 'fresh',
-          },
+      new FeedClient(process.env.POPULAR_FEED),
+      new FeedPreferencesConfigGenerator(
+        {},
+        {
+          includePostTypes: true,
+          includeBlockedSources: true,
```
Comment (rebelchris) [inline src/integrations/feed/generators.ts:118]: Does this mean we can more easily actually decide to block them now as well?
What was the reason we dont block them here?

Feel if you said you really hate a source and block it, it should not show anywhere basically 👀

Comment (idoshamun) [reply src/integrations/feed/generators.ts:118]: When it wasn't part of the feed service originally it was harder to do it.
Now with feed service everything is encapsulated and turned into config, so for api it's just turning on a flag :P
Pretty slick ah?

Comment (rebelchris) [reply src/integrations/feed/generators.ts:118]: 100% but we want to keep exposing blocked stuff there?

Comment (rebelchris) [reply src/integrations/feed/generators.ts:118]: Oh amazing! Misread it 🤧
Pretty cool 🙏

**Pull request merged**